README-Iframe.md

```diff
@@ -104,78 +101,8 @@ In this example, `data` is a `Map<String, String[]>` of the form data POSTed to
         // went wrong (incorrect session token, expired TTL, invalid signature)
     }
 
-### Separate Pairing and Authentication iframes
-
-#### Generating an Authentication iframe URI
-
-Every Toopher Authentication session should include a unique `requestToken` - a randomized `String` that is included in the signed request to the Toopher API and returned in the signed response from the Toopher `iframe`.  To guard against potential replay attacks, your code should validate that the returned `requestToken` is the same one used to create the request.
-
-Creating a random request token and storing it in the server-side session using the Java Servlet API:
-
-    private static final SecureRandom secureRandom = new SecureRandom();
-    // simple way to generate a randomized string.  
-    String requestToken = new BigInteger(20 * 8, secureRandom).toString(32);
-    request.getSession().setAttribute("ToopherRequestToken", requestToken);
-
-The Toopher Authentication API provides the requester a rich set of controls over authentication parameters.
-
-    String authIframeUrl = iframeApi.authUri(userName, resetEmail, actionName, automationAllowed, challengeRequired, requestToken, requesterMetadata, ttl);
-
-For the simple case of authenticating a user at login, a `loginIframeUrl` helper method is available:
-
-    String loginIframeUrl = iframeApi.loginUri(userName, resetEmail, requestToken)
-
-#### Generating a Pairing iframe URI
-
-    String pairIframeUrl = iframeApi.pairUri(userName, resetEmail)
+### Separate Pairing iframe
 
-#### Validating postback data from Authentication iframe and parsing API errors
+#### Generating a Pairing iframe URL
 
-In this example, `data` is a `Map<String, String[]>` of the form data POSTed to your server from the Toopher Authentication `iframe`.  You should replace the commented blocks with code appropriate for the condition described in the comment.
-
-    Map<String, String[]> data = httpServletRequest.getParameterMap();
-    String requestToken = (String)request.getSession().getAttribute("ToopherRequestToken");
-    // invalidate the Request Token to guard against replay attacks
-    request.getSession().removeAttribute("ToopherRequestToken");
-
-    try {
-        Map<String, String> validatedData = iframeApi.validate(data, requestToken);
-        if (validatedData.containsKey("error_code")) {
```

Comment:
I meant to move this up to Step 2: Validate the result. I'll update this doc and add this back in. Thanks for catching this.